src/Microsoft.Fx.Portability/ObjectModel/CompatibilityRangeAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+#if FEATURE_SERIALIZABLE
+    [Serializable]
+#endif
+    public class CompatibilityRangeAttribute
```

Comment:
Since this is no longer an attribute, you should rename it to `CompatibilityRange`.